sdk/formrecognizer/Azure.AI.FormRecognizer/MigrationGuide.md

```diff
@@ -8,7 +8,7 @@ Familiarity with `Azure.AI.FormRecognizer (3.1.x and below)` package is assumed.
 - [Migration benefits](#migration-benefits)
 - [Important changes](#important-changes)
     - [Client usage](#client-usage)
-    - [Analyzing document](#analyzing-documents)
+    - [Analyzing documents](#analyzing-documents)
```

Comment:
Up to you but I really liked the Terminology section that @witemple-msft wrote in his migration guide. We're adding this bit in the migration guide right under Important Changes.

https://github.com/Azure/azure-sdk-for-js/pull/17955#discussion_r720547576

Reply:
that is really cool!
Adding it